src/ralph/static/src/js/autofill.js

```diff
@@ -0,0 +1,27 @@
+(function ($) {
+    'use strict';
+    var connections = [
+        {
+            app: 'back_office', model: 'backofficeasset',
+            conf: [
+                {
+                    src: 'user',
+                    func: function($form, $src, val) {
+                        $('[name=owner]').val(val);
```

Comment:
what if i have matching input outside this form? shouldn't scope of this function be limited only to form? in other words, shouldn't you filter only inside passed form?